dask/dataframe/io/parquet/core.py

```diff
@@ -189,15 +190,51 @@ def read_parquet(
     read_from_paths : bool, default None
         Only used by ``ArrowDatasetEngine`` when ``filters`` are specified.
         Determines whether the engine should avoid inserting large pyarrow
-        (``ParquetFileFragment``) objects in the task graph. If this option
-        is True (default), ``read_partition`` will need to regenerate the
-        appropriate fragment object from the path and row-group IDs.  This will
-        reduce the size of the task graph, but will add minor overhead to
-        ``read_partition``.
-    chunksize : int, str
-        The target task partition size.  If set, consecutive row-groups
-        from the same file will be aggregated into the same output
-        partition until the aggregate size reaches this value.
+        (``ParquetFileFragment``) objects in the task graph.  If this option
+        is True, ``read_partition`` will need to regenerate the appropriate
+        fragment object from the path and row-group IDs.  This will reduce the
+        size of the task graph, but will add minor overhead to ``read_partition``.
+        By default (None), ``ArrowDatasetEngine`` will set this option to
+        ``False`` when there are filters.
+    chunksize : int or str, default None
+        The desired size of each output ``DataFrame`` partition in terms of total
+        (uncompressed) parquet storage space. If specified, adjacent row-groups
+        and/or files will be aggregated into the same output partition until the
+        cumulative ``total_byte_size`` parquet-metadata statistic reaches this
+        value. Use `aggregate_files` to enable/disable inter-file aggregation.
+    aggregate_files : bool or str, default None
+        Whether distinct file paths may be aggregated into the same output
+        partition. This parameter requires `gather_statistics=True`, and is
+        only used when `chunksize` is specified or when `split_row_groups` is
+        an integer >1. A setting of True means that any two file paths may be
+        aggregated into the same output partition, while False means that
+        inter-file aggregation is prohibited.
```

Comment:
Just to state the probably obvious. If this is simple `True` that is a greedy, `free-for-all` policy, i.e. no restrictions on what kind of files may be merged?

Reply:
Right. The following text is intended to say this: "A setting of True means that any two file paths may be aggregated into the same output partition" - Is this sufficient?